packages/orbit-components/src/ListChoice/ListChoice.stories.tsx

```diff
@@ -7,239 +9,185 @@ import RenderInRtl from "../utils/rtl/RenderInRtl";
 
 import ListChoice from ".";
 
+Button.displayName = "Button";
+ListChoice.displayName = "ListChoice";
+
 const getIcon = (source: string | null) => source && Icons[source];
 
-export default {
+const meta: Meta<typeof ListChoice> = {
   title: "ListChoice",
-};
-
-export const Default = ({ title, description, icon }) => {
-  const Icon = getIcon(icon);
-
-  return (
-    <ListChoice
-      title={title}
-      description={description}
-      icon={Icon && <Icon />}
-      onClick={action("onClick")}
-    />
-  );
-};
+  component: ListChoice,
 
-Default.story = {
   parameters: {
-    info: "Some description about this type of ListChoice in general.",
+    info: "Check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["onClick"],
+    },
   },
-};
 
-Default.args = {
-  title: "Choice Title",
-  description: "Further description",
-  icon: "Accommodation",
-};
+  args: {
+    title: "Choice Title",
+    description: "Further description",
+    icon: "Accommodation",
+    selectable: true,
```

Comment:
I am trying to figure out how to not have so many `undefined` in the file. Maybe `selectable` could be set to false here? You are setting it to undefined/false in multiple places so maybe let's just set it to `true` when relevant?

`selected` is also excluded in a lot of stories, maybe it can be not set on meta but only in Playground? Maybe there could be more optimisation being made with other args as well.

Reply:
> selected is also excluded in a lot of stories, maybe it can be not set on meta but only in Playground? Maybe there could be more optimisation being made with other args as well.

I thought we agreed at the start that we would define args in meta and then exclude for each of the stories where it's not needed.

Reply:
But that makes sense only when you have a lot of repeating args and it really makes sense to set it on meta so you are not repeating yourself. But if you set something on meta and then you are turning it off in every story, that doesn't really makes sense, no? :D